package.json

```diff
@@ -45,7 +45,8 @@
     "predocs-test": "npm run docs",
     "types": "dtsd bigquery v2 > ./src/types.d.ts",
     "prelint": "cd samples; npm link ../; npm install",
-    "precompile": "gts clean"
+    "precompile": "gts clean",
+    "export-types": "node scripts/exportTypes ./src ./src/index.ts types.d.ts index.ts && npm run fix src/index.ts"
```

Comment:
It's a little hard to know the what/why here.  Could I trouble you for a more detailed description on the PR describing the problem you're solving?